templates/serverpod_templates/projectname_server_upgrade/Dockerfile

```diff
@@ -1,29 +1,22 @@
-# If you update the dart version, make sure the image is
-# compatible with the busybox image.
 FROM dart:3.2.5 AS build
 
 WORKDIR /app
 COPY . .
 
 RUN dart pub get
-RUN dart compile exe bin/main.dart -o bin/main
+RUN dart compile exe bin/main.dart -o bin/server
 
-# If you update the busybox version, make sure the image is
-# compatible with the dart image.
-FROM busybox:1.36.1-glibc
-
-ENV runmode=development
-ENV serverid=default
-ENV logging=normal
-ENV role=monolith
+FROM scratch
 
 COPY --from=build /runtime/ /
-COPY --from=build /app/bin/main /app/bin/main
+COPY --from=build /app/bin/server server
 COPY --from=build /app/config/ config/
 COPY --from=build /app/web/ web/
```

Comment:
Suggestion: Copy over the migration files as well. Then we can close this PR: https://github.com/serverpod/serverpod/pull/2193

Reply:
This PR :) https://github.com/serverpod/serverpod/pull/2077 

We are currently also working on providing git credentials for accessing private repos when building on github:

https://github.com/serverpod/serverpod/pull/2124/files#diff-edf5e19a8e6e0c98a7cce72c93a5cfd6c9f0e74225af4824ae64b20d38b3d22c

Reply:
Awesome! ⭐